### ISSUE TITLE
Constant server disconnects when client email published

### ISSUE THREAD

**Issue by w111a** [FreeTAKTeam/FreeTakServer#162]
When I add an email in the ATAK setting " publish preferred email", my server will constantly disconnect stating "data reception timeout" and then reconnect. As soon as I remove the perferred email address, the server acts as expected. 

From FTS_info.log:

```WARNING : 2021-06-08 00:08:49,882 : Orchestrator.py:171 : there has been an error in a clients connectionerror in the creation of client information
WARNING : 2021-06-08 00:08:49,882 : Orchestrator.py:171 : there has been an error in a clients connectionerror in the creation of client information
WARNING : 2021-06-08 00:08:49,882 : Orchestrator.py:171 : there has been an error in a clients connectionerror in the creation of client information
WARNING : 2021-06-08 00:08:49,882 : Orchestrator.py:171 : there has been an error in a clients connectionerror in the creation of client information
WARNING : 2021-06-08 00:09:22,254 : ReceiveConnections.py:70 : error in Receive connections listen function timed out
INFO : 2021-06-08 00:09:29,846 : DataPackageServer.py:150 : sending client version info
INFO : 2021-06-08 00:09:29,848 : DataPackageServer.py:144 : sending client version json
INFO : 2021-06-08 00:09:30,899 : ReceiveConnections.py:54 : client connected
INFO : 2021-06-08 00:09:30,899 : Orchestrator.py:140 : client has connected
INFO : 2021-06-08 00:09:30,899 : Orchestrator.py:140 : client has connected
INFO : 2021-06-08 00:09:30,899 : Orchestrator.py:140 : client has connected
INFO : 2021-06-08 00:09:30,899 : Orchestrator.py:140 : client has connected
WARNING : 2021-06-08 00:09:30,903 : Orchestrator.py:171 : there has been an error in a clients connectionerror in the creation of client information
WARNING : 2021-06-08 00:09:30,903 : Orchestrator.py:171 : there has been an error in a clients connectionerror in the creation of client information
WARNING : 2021-06-08 00:09:30,903 : Orchestrator.py:171 : there has been an error in a clients connectionerror in the creation of client information
WARNING : 2021-06-08 00:09:30,903 : Orchestrator.py:171 : there has been an error in a clients connectionerror in the creation of client information
INFO : 2021-06-08 00:10:10,421 : DataPackageServer.py:150 : sending client version info
INFO : 2021-06-08 00:10:10,422 : DataPackageServer.py:144 : sending client version json
INFO : 2021-06-08 00:10:10,847 : ReceiveConnections.py:54 : client connected
INFO : 2021-06-08 00:10:10,847 : Orchestrator.py:140 : client has connected
INFO : 2021-06-08 00:10:10,847 : Orchestrator.py:140 : client has connected
INFO : 2021-06-08 00:10:10,847 : Orchestrator.py:140 : client has connected
INFO : 2021-06-08 00:10:10,847 : Orchestrator.py:140 : client has connected
WARNING : 2021-06-08 00:10:10,850 : Orchestrator.py:171 : there has been an error in a clients connectionerror in the creation of client information
WARNING : 2021-06-08 00:10:10,850 : Orchestrator.py:171 : there has been an error in a clients connectionerror in the creation of client information
WARNING : 2021-06-08 00:10:10,850 : Orchestrator.py:171 : there has been an error in a clients connectionerror in the creation of client information
WARNING : 2021-06-08 00:10:10,850 : Orchestrator.py:171 : there has been an error in a clients connectionerror in the creation of client information
WARNING : 2021-06-08 00:10:22,264 : ReceiveConnections.py:70 : error in Receive connections listen function timed out
INFO : 2021-06-08 00:10:50,531 : DataPackageServer.py:144 : sending client version json
INFO : 2021-06-08 00:10:50,533 : DataPackageServer.py:150 : sending client version info
INFO : 2021-06-08 00:10:50,873 : ReceiveConnections.py:54 : client connected
INFO : 2021-06-08 00:10:50,874 : Orchestrator.py:140 : client has connected
INFO : 2021-06-08 00:10:50,874 : Orchestrator.py:140 : client has connected
INFO : 2021-06-08 00:10:50,874 : Orchestrator.py:140 : client has connected
INFO : 2021-06-08 00:10:50,874 : Orchestrator.py:140 : client has connected
WARNING : 2021-06-08 00:10:50,879 : Orchestrator.py:171 : there has been an error in a clients connectionerror in the creation of client information
WARNING : 2021-06-08 00:10:50,879 : Orchestrator.py:171 : there has been an error in a clients connectionerror in the creation of client information
WARNING : 2021-06-08 00:10:50,879 : Orchestrator.py:171 : there has been an error in a clients connectionerror in the creation of client information
WARNING : 2021-06-08 00:10:50,879 : Orchestrator.py:171 : there has been an error in a clients connectionerror in the creation of client information
```

**Comment by brothercorvo** [comment]
probably another field missing in serialization @naman108

**Comment by brothercorvo** [comment]
fixed in 1.9.7
